hack/pre-run-e2e.sh

```diff
@@ -0,0 +1,267 @@
+  exit 1
+fi
+
+if [[ "${HC_SECRET_KEY}" = "" ]]; then
+  echo "HC_SECRET_KEY can not be empty, please set it, e.g.: export HC_SECRET_KEY=<your_sk>"
+  exit 1
+fi
+
+echo -e "\nBuild images"
+export REGISTRY_SERVER_ADDRESS=swr.ap-southeast-1.myhuaweicloud.com VERSION=latest
+make images
+
+echo -e "\nCheck cloud-config secret"
+count=`kubectl get -n kube-system secret cloud-config | grep cloud-config | wc -l`
+if [[ "$count" -ne 1 ]]; then
+  echo "Please create a secret with the name: cloud-config."
+  exit 1
+fi
+
+echo -e "\nDeploy huawei-cloud-controller-manager"
+cat << EOF | kubectl apply -f -
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:cloud-controller-manager
+rules:
+  - resources:
+      - tokenreviews
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+    apiGroups:
+      - authentication.k8s.io
+  - resources:
+      - configmaps
+      - endpoints
+      - pods
+      - services
+      - secrets
+      - serviceaccounts
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+    apiGroups:
+      - ''
+  - resources:
+      - nodes
+    verbs:
+      - get
+      - list
+      - watch
+      - delete
+      - patch
+      - update
+    apiGroups:
+      - ''
+  - resources:
+      - services/status
+      - pods/status
+    verbs:
+      - update
+      - patch
+    apiGroups:
+      - ''
+  - resources:
+      - nodes/status
+    verbs:
+      - patch
+      - update
+    apiGroups:
+      - ''
+  - resources:
+      - events
+      - endpoints
+    verbs:
+      - create
+      - patch
+      - update
+    apiGroups:
+      - ''
+  - resources:
+      - leases
+    verbs:
+      - get
+      - update
+      - create
+      - delete
+    apiGroups:
+      - coordination.k8s.io
+  - resources:
+      - customresourcedefinitions
+    verbs:
+      - get
+      - update
+      - create
+      - delete
+    apiGroups:
+      - apiextensions.k8s.io
+  - resources:
+      - ingresses
+    verbs:
+      - get
+      - list
+      - watch
+      - update
+      - create
+      - patch
+      - delete
+    apiGroups:
+      - networking.k8s.io
+  - resources:
+      - ingresses/status
+    verbs:
+      - update
+      - patch
+    apiGroups:
+      - networking.k8s.io
+  - resources:
+      - endpointslices
+    verbs:
+      - get
+      - list
+      - watch
+    apiGroups:
+      - discovery.k8s.io
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cloud-controller-manager
+  namespace: kube-system
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: system:cloud-controller-manager
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:cloud-controller-manager
+subjects:
+  - kind: ServiceAccount
+    name: cloud-controller-manager
+    namespace: kube-system
+---
+apiVersion: apps/v1
+kind: DaemonSet
```

Comment:
Why use `DaemonSet` instead of `Deployment`? Is there any specific reason?
AFAIK, we don't need to run it on each node.

Reply:
I will fix it in other PR